src/Symfony/Component/HttpClient/Response/TransportResponseTrait.php

```diff
@@ -231,7 +231,11 @@ public static function stream(iterable $responses, ?float $timeout = null): \Gen
                         } elseif ($chunk instanceof FirstChunk) {
                             if ($response->logger) {
                                 $info = $response->getInfo();
-                                $response->logger->info(\sprintf('Response: "%s %s"', $info['http_code'], $info['url']));
+                                $response->logger->info('Response: "{http_code} {url}" | Time: {total_time} seconds', [
```

Comment:
```suggestion
                                $response->logger->info('Response: "{http_code} {url}" | {total_time} seconds', [
```

I wonder if we really need this "Time: " prefix

Btw. is the unit seconds?

Reply:
Yes, it is seconds. I pushed up a change to remove the Time: prefix

I am open to suggestions on changing the format further.

Reply:
Thanks